canonical/loki-k8s-operator
# FIX #107 - Now we are using our statically compiled Promtail binary

## Issue

See #107 

## Solution


Now we are using our statically compilled promtail.

## Context

--

## Testing Instructions

1. Deploy this charm
2. Implement `LogProxyConsumer` in a consumer charm, for instance: https://github.com/jnsgruk/zinc-k8s-operator
3. Deploy consumer charm
4. Relate Loki with consumer charm
5. Verify that Promtail is running in consumer charm


## Release Notes
